hadoop-hdds/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockInputStream.java

```diff
@@ -152,7 +152,7 @@ protected int calculateExpectedDataBlocks(ECReplicationConfig rConfig) {
    * Using the current position, returns the index of the blockStream we should
    * be reading from. This is the index in the internal array holding the
    * stream reference. The block group index will be one greater than this.
-   * @return
+   * @return {@code int}
```

Comment:
This kind of description is redundant, return type is shown in method signature.  If no meaningful description can be added, we can remove the `@return` tag.

Reply:
Done